doc-content/jbpm-docs/src/main/asciidoc/Integration/integration-java-con.adoc

```diff
@@ -0,0 +1,12 @@
+[id='integration-java-con_{context}']
+= Integration with Java frameworks
+
+You can integrate
+ifdef::PAM,DM[]
+the {PROCESS_ENGINE}
+endif::PAM,DM[]
+ifdef::JBPM,DROOLS,OP[]
+jBPM
+endif::JBPM,DROOLS,OP[]
+with several industry-standard Java frameworks.
```

Comment:
Maybe to make this minimodule slightly more helpful, perhaps just add, "... Java frameworks, such as Apache Maven, CDI, Spring, and EJB." Just to leave the reader a little less hanging maybe. My two cents.